man/systemd.journal-fields.xml

```diff
@@ -104,6 +104,9 @@
           usually derived from glibc's
           <varname>program_invocation_short_name</varname> variable, see
           <citerefentry project='die-net'><refentrytitle>program_invocation_short_name</refentrytitle><manvolnum>3</manvolnum></citerefentry>.)</para>
+          <para>Journal does <emphasis>not</emphasis> validate any of the syslog fields.
```

Comment:
"Journal" → "The journal service".

Note that the journal actually doesn't validate any fields whatsoever that aren't prefixed with an underscore. This text suggests it would only exclude syslog related fields, while in fact there's no validation of any. Hence maybe cahnge to:

"Note that the journal service does not validate the values of any structured journal fields not prefixed with an underscore, and this includes any syslog-related fields such as these. Hence, applications that supply a facility, PID, or log level are expected to do so properly formatted, i.e. as numeric integers formatted as decimal strings."